h/static/scripts/test/admin-users-test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var AdminUsersController = require('../admin-users');
+
+// helper to create a native event
+function initEvent(eventType) {
+  // createEvent() used instead of Event constructor
+  // for PhantomJS compatibility
+  var event = document.createEvent('Event');
+  event.initEvent(eventType, true /* bubbles */, true /* cancelable */);
+  return event;
+}
+
+describe('AdminUsersController', function () {
+  var root;
+  var form;
+
+  beforeEach(function() {
+    root = document.createElement('div');
+    root.innerHTML = '<form id="js-users-delete-form">' +
+                     '<input type="submit" id="submit-btn">';
+    form = root.querySelector('form');
+    document.body.appendChild(root);
+  });
+
+  afterEach(function() {
+    root.parentNode.removeChild(root);
+  });
+
+  it('it submits the form when confirm returns true', function() {
+    var fakeWindow = {confirm: sinon.stub().returns(true)};
+    new AdminUsersController(root, fakeWindow);
+
+    var event = initEvent('submit');
+    var spy = sinon.spy(event, 'preventDefault');
```

Comment:
Rather than spying on the `preventDefault` method, you can use `event.defaultPrevented` - https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented , assuming that PhantomJS 1.x supports it. If not, this is fine.